bundle/metadata/annotations.yaml

```diff
@@ -6,10 +6,10 @@ annotations:
   operators.operatorframework.io.bundle.package.v1: windows-machine-config-operator
   operators.operatorframework.io.bundle.channels.v1: preview,stable
   operators.operatorframework.io.bundle.channel.default.v1: stable
-  operators.operatorframework.io.metrics.builder: operator-sdk-v1.32.0+git
+  operators.operatorframework.io.metrics.builder: operator-sdk-v1.32.0
```

Comment:
can remove this change

Reply:
May have asked this last time so apologies, but why is the `+git` there? 

Doesn't it mean someone used a dirty local branch of osdk and committed using that?

Reply:
agreed, looks like this has been in the code base for too long. But this is corrected when we do make bundle

Reply:
+1